### ISSUE TITLE
error run_tracker

### ISSUE THREAD

**Issue by pribadihcr** [bertinetto/siamese-fc#2]
Hi @bertinetto 

I got the following error:
>> run_tracker('demo-sequences/vot15-bag', true);
Warning: No stats found at /path/to/ILSVRC15-VID/stats.mat 
> In tracker at 49
  In run_tracker at 11 
No public field numChannels exists for class dagnn.BatchNorm.

Error in dagnn.Layer/load (line 195)
        obj.(f) = s.(f) ;

Error in dagnn.DagNN.loadobj (line 17)
    block.load(struct(s.layers(l).block)) ;

Error in load_pretrained (line 25)
    net = dagnn.DagNN.loadobj(net);

Error in tracker (line 53)
    net_z = load_pretrained([p.net_base_path p.net], p.gpus);

Error in run_tracker (line 11)
    tracker(params);

**Comment by bertinetto** [comment]
As explained in the main README, you should edit all the .example files and specify the absolute path to your directories. `/path/to/...` is just a filler to say that you have to specify your own path ;)

**Comment by pribadihcr** [comment]
Hi @bertinetto ,

I am sorry. It solved now. It is because I was using matconvnet-1.0-beta18.
Now I am using matlab2015b and matconvnet-1.0-beta20.
